AENS.md

```diff
@@ -148,6 +177,11 @@ interoperability with DNS.
 
 ***Note:*** For this first draft names MUST `.test` as a suffix, i.e. `mywallet.test`.
 
+Subnames are subject to the same restriction applied to names, with one
```

Comment:
It is discussible if we would not see google.test as a subname of the test domain. Clearly not, since it is a name, but the line is thin here